### ISSUE TITLE
DataPacketReceiveEvent not run

### ISSUE THREAD

**Issue by ProCOmpTEAM** [Hydreon/Steadfast2#447]
I have projects with forms. It normally work on another Pocketmine-MP branches. But this core not call this event.
For tests of debug I used this code, that not show into console messages. 
maybe I use something incorrect, becouse discuss about it with me. Thank.

My code
`public function onPacketReceived(DataPacketReceiveEvent $ev) {
		$pk = $ev->getPacket();
		$ev->getPlayer()->sendMessage("GOT $pk");
		if($pk instanceof ModalFormResponsePacket){
			$player = $ev->getPlayer();
			$formId = $pk->formId;
			$this->getServer()->getLogger->info("GOT ".$formId);
			$data = json_decode($pk->data, true);
			if(isset($this->forms[$formId])){
				/** @var Form $form */
				$form = $this->forms[$formId];
				if(!$form->isRecipient($player)){
					return;
				}
				$callable = $form->getCallable();
				if(!is_array($data)){
					$data = [$data];
				}
				if($callable !== null) {
					$callable($ev->getPlayer(), $data);
				}
				unset($this->forms[$formId]);
				$ev->setCancelled();
			}
		}
	}`
this code with formatting: [https://pastebin.com/QVpgNhqG](here)

latest version of core by 3.11

**Comment by GiantQuartz** [comment]
There isn't a "Form" class in Steadfast2.
Maybe you're using a wrong namespace for ModalFormResponsePacket, PMMP branches use a different namespace.

**Comment by debil9877654321** [comment]
No, it's bugged @GiantQuartz .. DataPacketReceiveEvent don't log any packets!

**Comment by debil9877654321** [comment]
It's not called anywhere in Player.php

**Comment by debil9877654321** [comment]
I found fix for this, but if I made PR they definitely don't merge it :P

**Comment by HerryYT** [comment]
actually isn't working .. so
